modules/ROOT/pages/_partials/endpoint-as2-send.adoc

```diff
@@ -14,7 +14,7 @@ AS2 Send Settings pertain to trading partner-specific AS2 settings such as `Cert
 
 |The HTTP(S) address at which your partner is accepting incoming AS2 messages.
 The hostname here is dependent upon how you set up your host in Mule Runtime.
-The port is automatically generated when you deploy a message flow that uses this endpoint.  
+The port is automatically generated when you deploy a message flow that uses this endpoint.
 This derived URL is what you give to your partners to send to with AS2. +
```

Comment:
Maria, it's confusing, we don't support AS2 Send yet. I think you want this to be an AS2 receive endpoint. If that's the case, then it should say "This derived URL is what you tell your partner is where you receive AS2 transmissions from them."

Reply:
Nope, I don't want to show this one.  I will hide this for now and add the AS2 Receive doc.

Reply:
New AS2 and SFTP receive files have been added.